fizz-cli/fizz_cli/utils.py

```diff
@@ -343,3 +327,91 @@ def rename_file(old_file_path, new_file_path):
 
     except Exception:
         return False
+
+def delete_function(function_name: str):
+    """
+    Deletes the function along with its associated files and configurations.
+
+    Parameters:
+        function_name (str): The name of the function to delete.
+
+    Returns:
+        bool: True if the function was successfully deleted, False otherwise.
+    """
+    try:
+        # Delete function YAML file
+        delete_file_if_exists(os.path.join(SPECS_DIR, f"function-{function_name}.yaml"))
+
+        # Delete route YAML file
+        delete_file_if_exists(os.path.join(SPECS_DIR, f"route-{function_name}.yaml"))
+
+        # Delete package YAML file
+        delete_file_if_exists(os.path.join(SPECS_DIR, f"package-{function_name}.yaml"))
+
+        # Delete function folder
+        shutil.rmtree(os.path.join(SPECS_DIR, function_name))
+
+        print(
+            f"[bold green]Function '{function_name}' and its associated files have been deleted successfully.[/bold green]"
+        )
+        return True
+    except Exception as e:
+        print(
+            f"[bold red]Error occurred while deleting function '{function_name}': {e}[/bold red]"
+        )
+        return False
+
+
+# Define Typer app
+app = typer.Typer()
+
+
+@app.command()
+def new(function_name: str):
+    """
+    Creates a new function with the given name.
+    """
+    print(f"Creating new function: {function_name} \n")
+
+
+@app.command()
+def rename(function_name: str, new_name: str):
```

Comment:
no app command should be here remove this shit

Reply:
done